app/templates/main/manageServiceLearningFaculty.html

```diff
@@ -34,7 +34,11 @@ <h2> Submitted </h2>
           <div class="col">
             <div class="float-end">
               <button type="button" class="btn btn-primary btn-sm">Send To Faculty Fellow</button>
-              <a type="button" href="/serviceLearning/newProposal" class="btn btn-success btn-sm">Add Course</a>
+            </div>
+            <div class = 'addNewCourse' align="right">
+              <form method="post" action="/serviceLearning/createCourse/" >
+                <button class="btn btn-success btn-sm" type = 'Submit' style='margin-right:16px'>Add Course </button>
```

Comment:
Put styles in a class instead of in a style attribute. That way we could use them in other places where we have two buttons next to each other